http.go

```diff
@@ -187,6 +188,7 @@ var mimeExtensions = map[string]string{
 	"war":     "application/java-archive",
 	"ear":     "application/java-archive",
 	"json":    "application/json",
+	"msgpack": "application/msgpack",
```

Comment:
This is not the correct mimetype

https://www.iana.org/assignments/media-types/application/vnd.msgpack